experimental/FTheoryTools/src/TateModels/constructors.jl

```diff
@@ -30,16 +45,68 @@ Normal toric variety
 
 julia> a1 = sum([rand(Int) * b for b in basis_of_global_sections(anticanonical_bundle(base))]);
 
+
 julia> a2 = sum([rand(Int) * b for b in basis_of_global_sections(anticanonical_bundle(base)^2)]);
+ERROR: MethodError: no method matching divisor_class(::ToricLineBundle)
+Closest candidates are:
+  divisor_class(!Matched::ToricDivisorClass) at ~/.julia/dev/Oscar/src/AlgebraicGeometry/ToricVarieties/ToricDivisorClasses/attributes.jl:20
+Stacktrace:
+ [1] ^(l::ToricLineBundle, p::ZZRingElem)
+   @ Oscar ~/.julia/dev/Oscar/src/AlgebraicGeometry/ToricVarieties/ToricLineBundles/constructors.jl:113
+ [2] ^(l::ToricLineBundle, p::Int64)
+   @ Oscar ~/.julia/dev/Oscar/src/AlgebraicGeometry/ToricVarieties/ToricLineBundles/constructors.jl:114
+ [3] literal_pow(f::typeof(^), x::ToricLineBundle, #unused#::Val{2})
+   @ Base ./intfuncs.jl:340
+ [4] top-level scope
+   @ none:1
 
```

Comment:
This commit obviously broke something in the FTheory code of Martin and Andrew......

Reply:
I was still messing with my git. The new branch should be okay now. I cleaned up the history.